ambry-api/src/main/java/com.github.ambry/account/Account.java

```diff
@@ -153,7 +153,7 @@ private Account(JSONObject metadata) throws JSONException {
    * @param status The status of the account. Cannot be null.
    * @param containers A collection of {@link Container}s to be part of this account.
    */
-  Account(Short id, String name, AccountStatus status, Collection<Container> containers) {
+  public Account(short id, String name, AccountStatus status, Collection<Container> containers) {
```

Comment:
why is this `public` now?

Reply:
changed it back to package private. Wanted to avoid requiring people to use the builder when it wasn't needed (when all fields were supplied), but that will create a lot of changes so i will do that in a future pr.